tests/test_optimizer.py

```diff
@@ -0,0 +1,32 @@
+from unittest import TestCase
+import numpy as np
+from chainer import cuda, Optimizer
+from chainer.optimizer import _sqnorm
+
+cuda.init()
+
+class TestOptimizerUtility(TestCase):
+    def setUp(self):
+        # x is an arithmetic progression of length 6
+        # whose common difference is 0.5
```

Comment:
I think this TODO comment should be placed where `_sqnorm` is defined, rather than in test code.